support-frontend/assets/helpers/forms/formValidation.ts

```diff
@@ -108,11 +130,10 @@ export const amountOrOtherAmountIsValid = (
 		selectedAmounts[contributionType] &&
 		selectedAmounts[contributionType] === 'other'
 	) {
-		if (
-			otherAmounts[contributionType] &&
-			otherAmounts[contributionType].amount
-		) {
-			amt = otherAmounts[contributionType].amount;
+		if (otherAmounts[contributionType].amount) {
```

Comment:
We've removed the check for `otherAmounts[contributionType]` being `undefined`, are we sure there's no way it could be `undefined`?

Reply:
Need to dig into this one again

Reply:
According to the type information it shouldn't be null but the `OtherAmounts` type had been changed into a Record by the initial migration